src/defaults/settings.js

```diff
@@ -84,5 +84,5 @@ export default //Renderer-specific settings
     gridlines: 'y',
     range_band: 24,
     margin: { top: 50 }, // for second x-axis
-    resizable: true
+    resizable: false
```

Comment:
why'd we change resizable to false?

Reply:
So that the multiples in the ID view can be resized manually.  This is necessary so that they don't overlap the right justified titles.